### PR TITLE
fix(learn): changed test to be different from the video

### LINKED ISSUE #39097: Numpy - Initialize Array Problem – challenge structure
**Describe your problem and how to reproduce it:**

In the numpy module, the [Numpy - Initialize Array Problem](https://www.freecodecamp.dev/learn/data-analysis-with-python/numpy/initialize-array-problem) challenge has a particular issue.

The challenge is introduced by the following line:

> What is another way to produce the following array?:

For accessibility, I though it would be best to add the original solution from the video. Something similar to 

---

As instructed in the video, the code:

```
output = np.ones((5, 5))
z = np.zeros((3, 3))
z[1, 1] = 9

output[1:-1, 1:-1] = z
```

Produces the following array

```
[[1. 1. 1. 1. 1.]
[1. 0. 0. 0. 1.]
[1. 0. 9. 0. 1.]
[1. 0. 0. 0. 1.]
[1. 1. 1. 1. 1.]]
```

What is another way to produce the same data structure?

---

However, the solution included in the challenge **is the one provided in the video**.

I'd be more than willing to make a pull request describing an alternative solution 👍 

**Add a Link to the page with the problem:**

https://www.freecodecamp.dev/learn/data-analysis-with-python/numpy/initialize-array-problem

### PR DESCRIPTION
**Pull request by nitnjain**
the array on the test and it's solution were same as the video explanation, replaced the test with one suggested in #39097

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #39097

Problem: Numpy - Initialize Array Problem [(link)](https://www.freecodecamp.org/learn/data-analysis-with-python/numpy/initialize-array-problem)

**Before fix:**
Question:
What is another way to produce the following array?
 ```
[[1. 1. 1. 1. 1.]
[1. 0. 0. 0. 1.]
[1. 0. 9. 0. 1.]
[1. 0. 0. 0. 1.]
[1. 1. 1. 1. 1.]]
```
Solution:
```
output = np.ones((5, 5))
z = np.zeros((3, 3))
z[1, 1] = 9
output[1:-1, 1:-1] = z
```

**After fix:**
Question:
What is another way to produce the following array?
```
[[0. 0. 0. 0. 0. 0. 0.]
[0. 1. 1. 1. 1. 1. 0.]
[0. 1. 1. 1. 1. 1. 0.]
[0. 1. 1. 5. 1. 1. 0.]
[0. 1. 1. 1. 1. 1. 0.]
[0. 1. 1. 1. 1. 1. 0.]
[0. 0. 0. 0. 0. 0. 0.]]
```
Solution:
```
output = np.ones((7, 7))
z = np.zeros((5, 5))
z[2, 2] = 5
output[1:-1, 1:-1] = z
```

### CODE REVIEW

File: curriculum/challenges/english/08-data-analysis-with-python/numpy/initialize-array-problem.md [hunk 1]
```diff
@@ -20,42 +20,44 @@ question:
...
+      output[1:1, -1:-1] = z
       ```
     - |
       ```py
-      output = np.ones((5, 5))
+      output = np.zeros((7,7))
 
-      z = np.zeros((3, 3))
-      z[1, 1] = 9
+      z = np.zeros((5, 5))
```
Comment (Sky020) [inline curriculum/challenges/english/08-data-analysis-with-python/numpy/initialize-array-problem.md:46]:
```suggestion
      z = np.ones((5, 5))
```

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
Thank you, for this PR.

The answer just needs this to be correct.

**Comment by Sky020** [comment]
@ojeytonwilliams and @Twaha-Rahman The Cyprus test for ShowCertification.js failed. Does this just mean this branch needs a rebase? Or, is there something else going on?

**Comment by Twaha-Rahman** [comment]
@Sky020 The fix for the Cypress test is in #39754.

A rebase after that PR gets merged should be enough.

**Comment by Sky020** [comment]
@Twaha-Rahman Right. Thank you. I just saw the message from Oliver on the contributors channel.

**Comment by RandellDawson** [comment]
Closing this PR due to not being able to push back to user's `master` branch.  I have created PR #39773 as a replacement with the user's changes.

@nitnjain In the future, please take a look at the docs to learn about creating a branch off your master for creating PRs.  You also should revert the changes made on this PR from your `master` branch.

**Pull request closed without merging**